freqtrade/tests/conftest.py

```diff
@@ -197,6 +197,71 @@ def default_conf():
     return configuration
 
 
+@pytest.fixture(scope="function")
+def all_conf():
```

Comment:
I think it might be better to load this from `config_full.json.example`. 
It's difficult enough to remember to put new options into one place - letalone two...

Reply:
How to do this better in the tests? Args with `-c config_full.json.example` and then `configuration.load_config()`?

Reply:
I don't think that'll work - this will do "too much" as it already validates the configuration, and you'll need to drop a few values first to test this properly.

best look at the test `test_load_config_file`  for a sample (without the mock)
Note that the variable says "validated conf" - but it is NOT validated yet.

``` python
configuration = Configuration(Namespace())
conf = configuration._load_config_file('somefile')
```